### DISCUSSION TITLE
mspider/user-guide/sidecar-management/passthrough/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1181] [Announcements]
# mspider/user-guide/sidecar-management/passthrough/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/sidecar-management/passthrough/

**Comment by mark8s** [comment]
流量透传的使用场景是什么？

**Reply by windsonsea** [reply]
Added some text to describe scenarios of passthrough in PR #5318
